Source/DafnyCore/DafnyCore.csproj

```diff
@@ -30,36 +29,8 @@
       <PackageReference Include="Boogie.ExecutionEngine" Version="2.15.7" />
   </ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\DafnyRuntime\DafnyRuntime.csproj" />
```

Comment:
I'm fine removing the reference to the DafnyRuntime, however, in [one of my PRs](https://github.com/dafny-lang/dafny/pull/2399), I am making use of the DafnyRuntime.cs which should then become a dependency of DafnyCore in your setting. Can we just add an external dependency to this file so that it's included at compile time?
That was we don't need a dependency of all the possible runtimes as you suggest but I have still it working for us.

Reply:
Your PR will actually work just fine after this change. Since you're not passing `/useRuntimeLib` when compiling your Dafny code through `dafny.msbuild`, the compiled output will have the embedded `DafnyRuntime.cs` file prepended, and you won't need to depend on `DafnyRuntime.dll`. 

That means that the version of the runtime you build your part of `DafnyCore` with doesn't affect the version that a `dafny` end user will end up depending on in their compiled code. It's admittedly a little weird and confusing to have the C# runtime code embedded like that though. :)

Reply:
Oh I see, it's in the generated code. That's fine I think ^^